src/test/java/uk/ac/ebi/embl/flatfile/reader/embl/EmblEntryReaderTest.java

```diff
@@ -979,6 +979,136 @@ public void testRead_IgnoreRepeatRegionWithLocasTag() throws IOException {
 		assertEquals(expectedEntryString, writer.toString());
 	}
 
+	public void testRead_IgnoreMissingCountry() throws IOException {
```

Comment:
-  testRead_IgnoreMissingValueTerms()

- a stronger test would loop all three qualifiers and all the missing values from IgnoreQualifier.MISSING_VALUE_TERMS and make sure they get removed.
-